fdbserver/storageserver.actor.cpp

```diff
@@ -974,6 +974,8 @@ ACTOR Future<Version> waitForVersionNoTooOld( StorageServer* data, Version versi
 ACTOR Future<Void> getValueQ( StorageServer* data, GetValueRequest req ) {
 	state int64_t resultSize = 0;
 	Span span("SS:getValue"_loc, { req.spanContext });
+	span.addTag(LiteralStringRef("key"), req.key);
+	span.addTag(LiteralStringRef("tag"), data->tag.toString());
```

Comment:
This is also a memory bug - the returning string is a temporary. I wonder whether having the tag here in the first place adds a lot of value. If you want to keep it, you should add a `Standalone<StringRef>` to the `StorageServer` struct to cache the value there (I think the `Tag` never changes in the lifetime of a storage?).

Reply:
The `key` tag is the important one, to able to identify traces. I'll remove `tag` here, it was mostly for testing purposes. I'm thinking about also removing some of the other metadata such as the `version` tags. We can always add them back later if they turn out to be helpful.